components/service/glean/README.md

```diff
@@ -3,6 +3,21 @@
 A client-side telemetry SDK for collecting metrics and sending them to Mozilla's telemetry service
 (eventually replacing [service-telemetry](../telemetry/README.md)).
 
+- [Before using the library](#before-using-the-library)
+- [Usage](#usage)
+    - [Setting up the dependency](#setting-up-the-dependency)
+    - [Integrating with the build system](#integrating-with-the-build-system)
+    - [Initializing glean](#initializing-glean)
+    - [Defining metrics](#defining-metrics)
+- [License](#license)
+
+## Before using the library
+Products using glean to collect telemetry **must**:
+
+- add documentation for any new metric collected with the library in its repository (see [an example](https://github.com/mozilla-mobile/android-components/commit/df429df1a193516f796f2330863af384cce820bc));
```

Comment:
This commit includes a number of things that aren't really relevant to what's being talked about here.  I assume you just mean the documentation chunk being added there?  Maybe we could link to that instead of the commit?

Reply:
I agree with @mdboom that it would be better to link to just the documentation example here.